CHANGES.md

```diff
@@ -5,6 +5,37 @@ all changes see the [closed
 milestones](https://github.com/cylc/cylc-flow/milestones?state=closed) for each
 release.
 
+-------------------------------------------------------------------------------
+## __cylc-8.0a2 (2019-Q4?)__
+
+Third alpha release of Cylc 8.
+
+### Backward incompatible changes
+
+The Cylc server program and CLI codebase is now a Python package that can be
+installed with `pip` (see #2990), and has been renamed to `cylc-flow`. The name
+`cylc` is now used as a native Python package namespace to allow other projects
+to re-use it and extend Cylc with plug-ins.
+
+Cylc 8.0a2 is not compatible with Cylc 7 or with Cylc 8.0a1, as the API is
+still under heavy development.
+
+The commands `cylc-profile-battery`, `cylc-test-battery`, `cylc-license`
```

Comment:
These next three paragraphs are exactly the same as in 8.0a1... should we leave them in only 8.0a1 and add new ones here as necessary? For 8.0 we can just collect all the BC incompatible changes and try to summarize them?

Reply:
(commenting here just because I was going to add a 8.0a2 changelog for the subscriptions PR but realized it would be simpler to wait for this PR to be merged first... out of the loop, so don't know what this PR is about, but happy to review if no other reviewer has spare time/knows about the code :+1: )

Reply:
I repeated this stuff for 8.0a2 because I figured most users will not actually see any of it until 8.0.0 is released.  But fair enough, it annoying and redundant to just duplicate the text.  How about I move it all up to the top, to become the eventual main change note for 8.0.0?

Reply:
>How about I move it all up to the top, to become the eventual main change note for 8.0.0?

Oh, that actually sounds like an even better idea. This way we can keep track of all backward compatibles changes coming for users migrating from 7.x. Which may save us a lot of time when it is time to release it.

Reply:
Done.

Reply:
(Tweaked right back to the cylc-8.0a0 change log blurb now.)